deploy.go

```diff
@@ -341,5 +342,19 @@ func (p *deploycmd) updateAppConfig(appf *appfile) error {
 	}
 
 	_, err := p.Apps.PatchAppsApp(param)
-	return err
+	if err != nil {
+		params := clientApps.NewPostAppsParams()
```

Comment:
so i learned yesterday we don't have a `PUT` for apps. so i will `PUT` my foot in my mouth. the 2.0 spec has PUT instead of POST everywhere (for this and other reasons) -- so anyway, ignore me for now, we don't need to go add PUT just for this, but would you mind leaving a note about ordering here and changing it after we do have PUT?

Reply:
Sure